wikitextprocessor/lua/_sandbox_phase2.lua

```diff
@@ -34,7 +34,13 @@ local function frame_args_index(new_args, key)
       key = i
    end
    local v = new_args._orig[key]
-   if v == nil then return nil end
+   if v == nil then
+       -- "inf" got converted to number type, convert back to string
```

Comment:
Could this error happen with names like "ind" (<- #IND?) or "nan"? Seems like this should fix all of them. This bug is *weird*!

Reply:
I have not idea how "inf" got converted to a number...

Reply:
Clarifying comment for people reading this in the future:
Apparently in (current?) Lua, you can use #INF and #IND to represent infinity (and I forget what #IND was), and the to_string return for the float value infinity is "inf" or "-inf" (and for NaN it's "nan", "-nan"). Maybe the name is evaluated as an expression (for some reason?), which turns it into a float somewhere?

Reply:
But Lua 5.1 doesn't have the `inf` variable, `loadstring("inf")` returns `nil`. Either `inf` was defined somewhere or `key` was converted to number by `tonumber()` before passing to `frame_args_index()`.

Reply:
Damn, tonumber("inf") returns the number `inf`.